docs/source/configuration/telemetry/instrumentation/selectors.mdx

```diff
@@ -75,15 +75,15 @@ The subgraph service executes multiple times during query execution, with each e
 |-----------------------------|-------------|------------------|--------------------------------------------------------------------------------|
 | `subgraph_operation_name`   | Yes         | `string`\|`hash` | The operation name from the subgraph query                                     |
 | `subgraph_operation_kind`   | No          | `string`         | The operation kind from the subgraph query                                     |
-| `subgraph_query`            | Yes         | `string`         | The graphql query to the subgraph                                              |
+| `subgraph_query`            | Yes         | `string`         | The GraphQL query to the subgraph                                              |
 | `subgraph_name`             | No          | `true`\|`false`  | The subgraph name                                                              |
```

Comment:
@michaelajsc noticed `subgraph_name` seems to suggest it should be a boolean, but `string` would make more sense here.

 @shorgi would you happen to know? (Slash could you be the second approver on this PR? We can fix the `subgraph_name` separately if need be.)

Reply:
It's indeed a boolean (see in [code](https://github.com/apollographql/router/blob/5421580ead8b2351b54d57d51be07ecb426335e9/apollo-router/src/plugins/telemetry/config_new/selectors.rs#L429)). Making the change here

```suggestion
| `subgraph_name`             | No          | `boolean`  | The subgraph name                                                              |
```